### DISCUSSION TITLE
5,6번 단계 질문

### DISCUSSION THREAD

**Discussion by rlrk** [changdoc/naver-vaccine-macro#148] [Q&A]
핸드폰 네이버앱으로 네이버 인증서 받고
컴퓨터로 특정병원 지정해서 자동백신예약 시도하려고 하는데요

알림신청 오기 전에 url 알아내는 방법으로 sid랑 orgCd 찾아서
https://v-search.nid.naver.com/reservation/standby?orgCd=XXXX&sid=XXXXX
여기에 그대로 넣었는데
https://v-search.nid.naver.com/reservation?orgCd=XXXX&sid=XXXXX
이 페이지로 연결되면서 잘못된 접근입니다 라고 떠서요ㅠㅠ...
(두 주소의 XXXX와 XXXXX는 서로 같은 값입니다)

혹시 뭘 잘못했는지.... 도움 부탁드립니다 ㅠㅠ

참고로 사용하고 있는 버전은
https://github.com/changdoc/naver-vaccine-macro/issues/29
여기서 받은 최신버전입니다.

**Comment by changdoc** [comment]
#4 내용 다시 한번 보세요~

**Reply by changdoc** [reply]
orgCd와 sid는 다른 값입니다.

**Reply by rlrk** [reply]
네 서로 다른 값 찾아서 넣었습니다 ㅠㅠ

**Reply by rlrk** [reply]
#4 가서 보니 13일 전에 예약페이지 막아 놓은 것 같다고 코멘트가 달려 있던데 이제는 그 방법으로 안되는 건가요?

**Reply by changdoc** [reply]
예전에는 orgCd만으로도 접근이 되었는데 현재는 sid까지 넣어야 접근이 되는 상황입니다.

**Reply by rlrk** [reply]
https://v-search.nid.naver.com/reservation/standby?orgCd=XXXX&sid=XXXXX
여기서 id에서 찾은 값 sid에 넣고 vaccineOrganizationCode에서 찾은 값 orgCd에 넣고
그 주소 그대로 새 크롬창에 넣고 페이지를 열었는데
잘못된 접근이라고 떠서요..

**Reply by changdoc** [reply]
만드신 주소 여기에 입력해보시겠어요?

**Reply by changdoc** [reply]
https://v-search.nid.naver.com/reservation?orgCd=31341560&sid=19513483

https://v-search.nid.naver.com/reservation/standby?orgCd=31341560&sid=19513483

이 링크가 설명대로 만든 주소인데요. 잘 이동 되는거 보면 데이터가 잘 못 된 상태일듯하네요.

**Reply by rlrk** [reply]
orgCd= 을 살려두고 XXXX 자리에 제가 찾은 숫자를 넣으면 되는 건가요? 저는 orgCd를 지우고 넣었었거든요;;

**Reply by changdoc** [reply]
XXX부분을 숫자로 바꾸는거니 orgCd=는 남아있어야 합니다

**Reply by rlrk** [reply]
아하!!! 감사합니다!!! 해결 되었습니다ㅠㅠ!!! 초보라서 orgCd 자리를 대체하는 건줄 알았어요 ㅠㅠ 정말 감사합니다!!!
